Source/DedicatedServer/Private/ServerConsoleLinux.cpp

```diff
@@ -1,23 +1,244 @@
 // Copyright 2004-2016 YaS-Online, Inc. All Rights Reserved.
 
 #include "DedicatedServerPrivatePCH.h"
+#if PLATFORM_LINUX
+#include <iostream>
+#include <sstream>
+#include <string>
```

Comment:
Yes, sorry about that, but I've searched and there's not way to correctly format preprocessor directives with standard Visual Studio. Maybe there's a VS plugin for that, but I don't know too much about this. As I've found, the preprocessor directives are formatted to the first column because in older compilers, the code would haven't been compiled if there were a space before them.

Reply:
Yeah, I've been constantly fighting with VS's formatter, to the point that I only allow formatting to happen on paste
Eclipse and IDEA have way more flexible formatter solutions, even VSCode lacks it for the most part
And ClangFormat is something that I haven't used in production yet.